conf.py

```diff
@@ -133,3 +133,5 @@
 if 'github_issues' in html_context and html_context['github_issues'] and not disable_feedback_button:
     html_js_files.append('github_issue_links.js')
 html_js_files.extend(custom_html_js_files)
+
+generate_ams_configuration()
```

Comment:
Minor nitpick: Can we add comments explaining this is an Anbox specific thing and that we need to add this line to conf.py everytime we upgrade to a new version of starter pack?

Ideally, we are never supposed to edit conf.py and if edits are needed, we are supposed to push it upstream. But this is such a project specific thing that I don't see value in pushing this to upstream. 
```suggestion
# Anbox specific function to generate dynamic AMS configuration
# Add this change to conf.py every time the starter pack is upgraded to a later version.
generate_ams_configuration()
```